meinberlin/apps/contrib/assets/__tests__/CountDown.jest.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { CountDown } from '../CountDown'
+
+test('CountComponent is showing inactive', () => {
+  render(
+    <CountDown
+      activeClass="active-class"
+      inactiveClass="inactive-class"
+      countText="votes"
+      counter={0}
+    />
+  )
+  const countComponent = screen.getByText('votes')
+  expect(countComponent.innerHTML).toBe('votes')
+})
+
+test('CountComponent is showing active', () => {
```

Comment:
these two tests seem extremely similar, would it maybe make sense to do a `getByText('0')` and `getByText('2')` also check that the count is correct?

Reply:
cool, thanks